package.json

```diff
@@ -146,7 +146,8 @@
     "kind-of": "6.0.3",
     "minimist": "1.2.8",
     "merge": "^2.1.1",
-    "normalize-url": "^8.0.0"
+    "normalize-url": "^8.0.0",
+    "baseline-browser-mapping": "2.9.9"
```

Comment:
We don't need to add it as a new dependency just for browserslist. Instead, a rather simpler way in this case could be to remove the baseline-browser-mapping from yarn.lock and run `yarn install`. It should automatically pick the latest version of this package.